docs/source/monitors.rst

```diff
@@ -6,6 +6,7 @@ Monitoring your jobs
 
 Monitors
 --------
+.. _`Monitors`
```

Comment:
This is malformed.

See the output of `make html` within the `docs` folder (be warned: you’ll also get existing issues unrelated to your changes). You first need to install the dependencies from `requiremets-docs.txt`, on the root folder.

You don’t need a mark at all, actually. [Section titles should already work as targets](https://stackoverflow.com/a/18704682/939364).

Same applies to the section below.

Reply:
Fixed.